docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss

```diff
@@ -162,6 +162,11 @@
 
       h2 {
         @include hb-heading-4;
+        margin-top: hb-calculate-rems(24px);
+
+        @include grid-media-min('md') {
```

Comment:
I think this might look better if it switched at  `grid-media-min('lg')` (since that's when the hb-heading-4 font changes sizes). When it switches at `md`, we get this middle-zone where the font is smaller but the margin-top is still pretty big:

Small:
<img width="401" alt="Screen Shot 2020-03-25 at 9 16 42 PM" src="https://user-images.githubusercontent.com/1256329/77600326-f6765780-6edd-11ea-9642-9d6a8ee01230.png">

Middle-zone:
<img width="532" alt="Screen Shot 2020-03-25 at 9 14 51 PM" src="https://user-images.githubusercontent.com/1256329/77600263-badb8d80-6edd-11ea-98a9-02f0899e867b.png">

Large:
![image](https://user-images.githubusercontent.com/1256329/77600280-c8911300-6edd-11ea-98c4-a498995a78d3.png)

Reply:
good catch!!!